### DISCUSSION TITLE
How can I define the custom view of MultifileElement that shows the mockup image for video on 'gallery'?

### DISCUSSION THREAD

**Discussion by dovelive** [vueform/vueform#307] [Ideas]
Now, when I set the view value of MultifileElement to 'gallery', thumbnail images appear correctly when I select an image file.
However, when I choose a video file, I see a broken thumbnail image.
I’d like to display a mockup video icon instead when a video file is selected.
I think I might need to create a custom template for this, but I'm not sure how to go about it.
Can anyone help me implement this?

**Comment by adamberecz** [comment]
Here's an example for doing it with PDF:
https://vueform.com/recipes/pdf-preview-for-files

You can use the same method to create a custom view for videos.

Or, if you don't want to bother with it, it might be an option to replace just the CSS class and/or define some custom CSS.
